src/semantic-ui/site/elements/container.overrides

```diff
@@ -268,3 +268,16 @@ Container Overrides - REACT-INVENIO-APP-ILS
     }
   }
 }
+
+.ui.container.link-like {
+    color: #4183C4 !important;
```

Comment:
if you hardcode the color like this, nobody will be able to override it in the overlay application (f.e. cds-ils)

Reply:
I am not sure how this is influencing the fact the color is hardcoded. You should be able to read it from one of the available LESS variables